.travis.yml

```diff
@@ -13,6 +13,9 @@ services:
 
 before_script: |
   cp config/database.example.yml config/database.yml
+  cp config/vsphere.example.yml config/vsphere.yml
```

Comment:
could be provider_vsphere.yml, because in future we could have more systems and this could visually group them together